.pre-commit-config.yaml

```
@@ -19,7 +19,7 @@ repos:
   - id: fix-encoding-pragma
 
 - repo: https://github.com/charliermarsh/ruff-pre-commit
-  rev: "v0.0.269"
+  rev: "v0.0.270"
```

```suggestion
  rev: "v0.0.271"
```

Might as well go all the way. :)